src/Board.ts

```diff
@@ -90,17 +91,26 @@ export function canMoveMarbleInDirection(
   marblePosition: string,
   direction: string
 ): Boolean {
-  return (
-    positionExistsInBoard(boardGraph, marblePosition) &&
-    hasFreeSpotBeforeToMove(boardGraph, marblePosition, direction)
-  );
+  
+    const existInBoard = positionExistsInBoard(boardGraph, marblePosition);
+    if (!existInBoard) {
+        throw new CantMoveError('This position does not exist in board')
+    }
+
+    const freeSpotBeforeToMove = hasFreeSpotBeforeToMove(boardGraph, marblePosition, direction);
+    if (!freeSpotBeforeToMove) {
+      throw new CantMoveError("This marble can't move in this direction")
+    }
+
+    return existInBoard && freeSpotBeforeToMove;
```

Comment:
why do you return a boolean value if you throw already exceptions when it is false?